index.html

```diff
@@ -63,8 +63,117 @@ <h3 class="skill-title">Skills</h3>
         <div>
             <div class="eli18"><img src="images/icones/Ellipse18.svg" alt=" img ackround "></div>
             <div class="eli19"><img src="images/icones/Ellipse19.svg" alt="img backgound "></div>
+  
         </div>
     </section>
+    <div class="work">
+        <h3>WORK</h3>
+      </div>
+    <div class="recent-work">
+      <h2 class="recent">My recent <br> <br>Work</h2>
+    </div>
+  <hr>
+  <section class="project" id="aboutproject">
+    <div class="first-project child">
```

Comment:
As the project requirement, the main container of the projects has to be `display: grid` and you can use `Flexbox` for the specific cards.

<img width="1028" alt="image" src="https://user-images.githubusercontent.com/34578631/171254940-d796fe8a-2933-46f9-94b8-e69cda97af97.png">